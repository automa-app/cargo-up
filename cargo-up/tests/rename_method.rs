use insta::assert_snapshot;
mod utils;

#[test]
fn test_rename_method() {
    let (out, err) = utils::run_upgrader("rename_method", "0.3.0", true);
    assert_snapshot!(out);
    assert_snapshot!(err);
}
