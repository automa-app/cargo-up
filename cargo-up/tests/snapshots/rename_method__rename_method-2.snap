---
source: tests/rename_method.rs
expression: err
---
Pre loading upgradee ... done

